zerver/tests/test_scheduled_messages.py

```diff
@@ -389,7 +390,7 @@ def test_editing_failed_send_scheduled_message(self) -> None:
             response = self.client_patch(
                 f"/json/scheduled_messages/{scheduled_message.id}", payload_without_timestamp
             )
-            self.assert_json_error(response, "Scheduled delivery time must be in the future.")
+            self.assert_json_error(response, DeliveryTimeNotInFutureError.msg_format())
```

Comment:
Better to keep this as the string -- tests are one place where end-to-end verification is more important than avoiding duplication of this form.

(If we have a bug where `msg_format` returns garbage, this changed version can't catch it!)

Reply:
Undid these 3 hunks before merging.